src/backend/projects/project_crud.py

```diff
@@ -19,19 +19,37 @@
 
 import geojson
 import io
-import json
+import os
+from json import loads, dumps, dump
 from typing import List
 from zipfile import ZipFile
+import epdb
 from fastapi import HTTPException, UploadFile
 from geoalchemy2.shape import to_shape
-from geojson_pydantic import FeatureCollection
-from shapely.geometry import mapping, shape
+import geoalchemy2
+import numpy as np
+from pyxform.xls2xform import get_xml_path, xls2xform_convert
+import xmltodict
+
+
+from geojson_pydantic import FeatureCollection, Feature
+from shapely.geometry import mapping, shape, Polygon
```

Comment:
I strongly suspect we'll need to replace this with PostGIS functionality, but for now it's fine to get things into basic working shape.

Reply:
Actually some geometry manipulation needs to be done outside of postgis.

Reply:
Really? I'm surprised by that; as far as I can see PostGIS can do pretty much any geoprocessing you like, and make you a tuna sandwich as well. If this is the case it has implications for my splitting algorithm bounty project...